src/native/util/common.h

```diff
@@ -105,7 +105,7 @@ class Exception : public std::exception
 
     virtual const char* what() const throw()
     {
-        return (std::string("Exception: ") + _msg).c_str();
+        return "test";
```

Comment:
I tested and this works as the quickest fix:
```suggestion
        return _msg.c_str();
```